libdevcore/Log.h

```diff
@@ -268,6 +261,8 @@ class LogOutputStream: LogOutputStreamBase
 
 	LogOutputStream& operator<<(LogTag _t) { m_logTag = _t; return *this; }
 
+	LogOutputStream& operator<<(const char* _t) { if (Id::verbosity <= g_logVerbosity) { if (_AutoSpacing && m_sstr.str().size() && m_sstr.str().back() != ' ') m_sstr << " "; append(_t); } return *this; }
```

Comment:
Do we need this? Wouldn't `append` overload for `const char*` above be enough for this?